src/main/java/bio/terra/workspace/service/iam/model/IamRole.java

```diff
@@ -1,23 +1,24 @@
 package bio.terra.workspace.service.iam.model;
 
+import bio.terra.workspace.generated.model.ApiIamRole;
 import java.util.Arrays;
 import java.util.Optional;
 
 /** Internal representation of IAM roles. */
 public enum IamRole {
-  READER("reader", bio.terra.workspace.generated.model.IamRole.READER),
-  WRITER("writer", bio.terra.workspace.generated.model.IamRole.WRITER),
-  OWNER("owner", bio.terra.workspace.generated.model.IamRole.OWNER);
+  READER("reader", bio.terra.workspace.generated.model.ApiIamRole.READER),
```

Comment:
You don't need the qualification anymore here.

Reply:
Thanks